app/main/views/suppliers.py

```diff
@@ -444,3 +443,28 @@ def create_your_account_complete():
         "suppliers/create_your_account_complete.html",
         email_address=email_address
     ), 200
+
+
+@main.route('/<framework_slug>/opportunities-overview', methods=['GET'])
+@login_required
+def opportunities_overview(framework_slug):
```

Comment:
I can't remember exactly why but I think it came out of discussion with @laurenceberry? Potentially something to do with how the supplier page is going to be designed in the future and this is a step towards that.